pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "brevurl"
+version = "0.1.0"
+description = "Open source, fast and modern URL shortener"
+authors = ["Your Name <you@example.com>"]
```

Comment:
Please add your own name in the "authors" section, you can also add an e-mail later.